apps/portlet-explorer/src/main/webapp/groovy/webui/component/explorer/versions/UIVersionInfo.gtmpl

```diff
@@ -55,25 +55,9 @@ import java.util.Locale;
   	def context = _ctx.getRequestContext();
   	def jsManager = context.getJavascriptManager();
 
-    String statusTitle = StringEscapeUtils.escapeJavaScript(_ctx.appRes("UserProfilePopup.label.Loading"));
-    String connectLabel = StringEscapeUtils.escapeJavaScript(_ctx.appRes("UserProfilePopup.label.Connect"));
-    String confirmLabel = StringEscapeUtils.escapeJavaScript(_ctx.appRes("UserProfilePopup.label.Confirm"));
-    String cancelRequestLabel = StringEscapeUtils.escapeJavaScript(_ctx.appRes("UserProfilePopup.label.CancelRequest"));
-    String removeConnectionLabel = StringEscapeUtils.escapeJavaScript(_ctx.appRes("UserProfilePopup.label.RemoveConnection"));
-    String ignoreLabel = StringEscapeUtils.escapeJavaScript(_ctx.appRes("UserProfilePopup.label.Ignore"));
-
-    String labels = """ {
-    StatusTitle: '${statusTitle}',
-    Connect: '${connectLabel}',
-    Confirm: '${confirmLabel}',
-    CancelRequest: '${cancelRequestLabel}',
-    RemoveConnection: '${removeConnectionLabel}',
-    Ignore: '${ignoreLabel}'
-    } """;
-
     def actionCompareLink = StringUtils.getNestedString(uicomponent.event("CompareVersion"),"javascript:ajaxGet(\'","\')");
     jsManager.require("SHARED/UIVersionInfo", "uiVersionInfo").addScripts("uiVersionInfo.init(\""+uicomponent.id+"\");")
-           .addScripts("uiVersionInfo.initUserProfilePopup($labels);")
+           .addScripts("uiVersionInfo.initUserProfilePopup({});")

```

Comment:
Can you make the labels empty object `{}` not mandatory so that we can call `initUserProfilePopup` without parameters. 

```suggestion
           .addScripts("uiVersionInfo.initUserProfilePopup();")
```

Reply:
@rdenarie can you please apply this on all other calls of `initUserProfilePopup`

Reply:
done